src/compiler/utilities.ts

```diff
@@ -1087,6 +1100,10 @@ module ts {
         return SyntaxKind.FirstTriviaToken <= token && token <= SyntaxKind.LastTriviaToken;
     }
 
+    export function isAsyncFunctionLike(node: Node): boolean {
+        return isFunctionLike(node) && !isAccessor(node) && (node.flags & NodeFlags.Async) === NodeFlags.Async;

```

Comment:
Accessors cannot be marked async. Since I use this test to branch emit I don't want to accidentally emit the async-specific wrappers in an accessor if they happened to put the `async` keyword on an accessor, even if we have an error reported from the checker.